broccoli/transforms/inject-node-globals.js

```diff
@@ -6,7 +6,9 @@ function injectNodeGlobals({ types: t }) {
     name: 'inject require',
     visitor: {
       Program: {
-        enter(path) {
+        enter(path, state) {
+          if (state.file.opts.filename.match(/node-module\/index.js/)) return;
```

Comment:
`filename.endsWith('node-module/index.js')` might be easier to read:

```suggestion
          if (state.file.opts.filename.endsWith('node-module/index.js')) return;
```

Reply:
Can followup with this later, I'd like to get it merged for now